tests/integration/helm_otlp_test.go

```diff
@@ -13,7 +13,7 @@ func Test_Helm_OTLP(t *testing.T) {
 
 	expectedMetrics := internal.DefaultExpectedMetrics
 	// we have tracing enabled, so check tracing-specific metrics
-	expectedMetrics = append(expectedMetrics, internal.TracingOtelcolMetrics...)
+	expectedMetrics = append(expectedMetrics, internal.TracingOtelcolMetrics..., internal.FileConsumerOtelColMetrics...)
```

Comment:
In https://github.com/SumoLogic/sumologic-kubernetes-collection/actions/runs/10107195658/job/27950661632 `Set test names`

```
Run echo "test_names_allversions=$(make list-tests-allversions | jq -R . | jq -cs .)" >> $GITHUB_OUTPUT
  echo "test_names_allversions=$(make list-tests-allversions | jq -R . | jq -cs .)" >> $GITHUB_OUTPUT
  echo "test_names_onlylatest=$(make list-tests-onlylatest | jq -R . | jq -cs .)" >> $GITHUB_OUTPUT
  shell: /usr/bin/bash -e {0}
  env:
    KIND_VERSION: 0.17.0
  
# github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration
Error: helm_ot_default_test.go:16:79: expected ')', found internal
# github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration
Error: helm_otlp_test.go:16:79: expected ')', found internal
```